### PR TITLE
feat/plugin: inspire us

### PR DESCRIPTION
**Pull request by SteveParson**
The MUN CS Society discord server is not inspirational enough. We need to seed it with inspirational quotes.

### CODE REVIEW

File: plugins/WiseQuote/plugin.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "WiseQuote",
+  "version": "1.0.0",
+  "author": "Bartha",
```
Comment (jackharrhy) [inline plugins/WiseQuote/plugin.json:4]: this does not seem to be the true author

File: plugins/WiseQuote/__init__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        quote = random.choice(self.quotes)
+        response = '"{text}" -**{author}**'
+        await ctx.send(response.format(
+            text = quote['text'],
+            author = quote['author'] if 'author' in quote.keys() and quote['author'] != None else 'Unknown')
+        )
+
+    def __init__(self, manifest, bot):
+        super().__init__(manifest, bot)
+        self.quotes = json.loads(requests.get(QUOTES_ENDPOINT).text)
```
Comment (jackharrhy) [inline plugins/WiseQuote/__init__.py:23]: `__init__` before `wisequote`, usually `__inti__` goes first

**Review by jackharrhy — CHANGES_REQUESTED** [review]
amazing plugin

some comments

File: plugins/WiseQuote/__init__.py (hunk 2 above)
Comment (SteveParson) [reply plugins/WiseQuote/__init__.py]: @jackharrhy yesss

File: plugins/WiseQuote/plugin.json (hunk 1 above)
Comment (SteveParson) [reply plugins/WiseQuote/plugin.json]: I've added the first initial as per the true author's request.

**Review by jackharrhy — APPROVED** [review]

**Pull request merged**